core/src/test/java/io/aiven/kafka/tieredstorage/RemoteStorageManagerMetricsTest.java

```diff
@@ -105,38 +105,50 @@ void metricsShouldBeReported() throws RemoteStorageException, JMException {
 
         final InputStream resultInputStream = rsm.fetchLogSegment(REMOTE_LOG_SEGMENT_METADATA, 0);
 
-        final ObjectName segmentCopyPerSecName = ObjectName.getInstance(
+        final ObjectName rsmMetricsName = ObjectName.getInstance(
             "aiven.kafka.server.tieredstorage:type=remote-storage-manager-metrics");
-        assertThat((double) MBEAN_SERVER.getAttribute(segmentCopyPerSecName, "segment-copy-rate"))
+        assertThat((double) MBEAN_SERVER.getAttribute(rsmMetricsName, "segment-copy-rate"))
             .isEqualTo(3.0 / METRIC_TIME_WINDOW_SEC);
-        assertThat((double) MBEAN_SERVER.getAttribute(segmentCopyPerSecName, "segment-copy-total"))
+        assertThat((double) MBEAN_SERVER.getAttribute(rsmMetricsName, "segment-copy-total"))
             .isEqualTo(3.0);
 
-        assertThat((double) MBEAN_SERVER.getAttribute(segmentCopyPerSecName, "segment-copy-time-avg"))
-            .isZero();
-        assertThat((double) MBEAN_SERVER.getAttribute(segmentCopyPerSecName, "segment-copy-time-max"))
-            .isZero();
-
-        assertThat((double) MBEAN_SERVER.getAttribute(segmentCopyPerSecName, "segment-copy-bytes-rate"))
+        assertThat((double) MBEAN_SERVER.getAttribute(rsmMetricsName, "segment-copy-bytes-rate"))
             .isEqualTo(30.0 / METRIC_TIME_WINDOW_SEC);
-        assertThat((double) MBEAN_SERVER.getAttribute(segmentCopyPerSecName, "segment-copy-bytes-total"))
+        assertThat((double) MBEAN_SERVER.getAttribute(rsmMetricsName, "segment-copy-bytes-total"))
```

Comment:
Do we really need to cast to double everywhere? `isZero` will not work without cast but `isEqualTo(0.0)` will in case you want to keep common style throughout the method.